### PR TITLE
Install webpacker

### PR DESCRIPTION
**Pull request by aaron-collier**
## Why was this change made?

Initial installation of webpacker to setup ability to add UI components.

## Was the documentation (README, DevOpsDocs, wiki, openapi.json) updated?

N/A

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -32,6 +32,8 @@ gem 'puma', '~> 3.11'
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.1.0', require: false
 
+gem 'webpacker', '~> 4.x'
```
Comment (jcoyne) [inline Gemfile]: this doesn't look like a valid gem specification.

Comment (aaron-collier) [reply Gemfile]: Huh, that was copied from https://github.com/rails/webpacker, should this just be webpacker without the version pin?

Comment (mjgiarlo) [reply Gemfile]: I suspect @jcoyne was reacting to the literal `x`. If it works, I'm OK with it. Though perhaps to be consistent with our other `Gemfile`s, we could change it to `'~> 4.0'`

Comment (aaron-collier) [reply Gemfile]: Gotcha @mjgiarlo done.

**Pull request merged**